analytics_data_api/v0/views/videos.py

```diff
@@ -10,19 +10,20 @@
 
 class VideoTimelineView(generics.ListAPIView):
     """
-    Get the timeline for a video.
+    Get the counts of users and views for a video.
 
-    **Example request**
+    **Example Request**
 
         GET /api/v0/videos/{video_id}/timeline/
 
     **Response Values**
 
-        Returns viewing data for segments of a video.  Each collection contains:
+        Returns viewing data for each segment of a video.  For each segment,
+        the collection contains the following data.
```

Comment:
Segments could be 5 seconds and is typically, but the segment length is defined in the videos endpoint.